### PR TITLE
the CSSStyleSheet.href is not overridden (close #593)

### LINKED ISSUE #593: the CSSStyleSheet.href is not overridden

### PR DESCRIPTION
**Pull request by churkin**
\cc @kirovboris @miherlosev

### CODE REVIEW

**Comment by churkin** [comment]
FPR

File: src/client/sandbox/code-instrumentation/properties/index.js [hunk 1]
```diff
@@ -173,16 +173,33 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
 
             href: {
                 condition: el => {
+                    if (LocationAccessorsInstrumentation.isLocationWrapper(el))
```
Comment (miherlosev) [inline src/client/sandbox/code-instrumentation/properties/index.js]: Change to 
```javascript
if (LocationAccessorsInstrumentation.isLocationWrapper(el) || isStyleSheet(el))
    return true;
else if (domUtils.isDomElement(el))
  return URL_ATTR_TAGS['href'].indexOf(domUtils.getTagName(el)) !== -1;

return false
```

Comment (churkin) [inline src/client/sandbox/code-instrumentation/properties/index.js]: This changes the order of checks. In the beginning have to go most likely and fast checks.

Comment (miherlosev) [inline src/client/sandbox/code-instrumentation/properties/index.js]: Ok

**Comment by miherlosev** [comment]
Then lgtm

**Comment by kirovboris** [comment]
lgtm

**Pull request merged**